internal/config/config.go

```diff
@@ -1,16 +1,21 @@
 package config
 
+// Config holdsthe entire application configuration.
```

Comment:
Minor typo 'holds the' instead of 'holdsthe'

Reply:
Done! Sorry for the delay.